### PR TITLE
Events

### PR DESCRIPTION
**Pull request by tonnymuchui**
- Use reacts events :heavy_check_mark: 
- Use React life cycle methods :heavy_check_mark: 
-

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -21,29 +33,29 @@ class Calculator extends Component {
...
-            <button type="button" className="press-btn">1</button>
-            <button type="button" className="press-btn">2</button>
-            <button type="button" className="press-btn">3</button>
-            <button type="button" className="press-btn amber">+</button>
-
-            <button type="button" className="press-btn-2">0</button>
-            <button type="button" className="press-btn">.</button>
-            <button type="button" className="press-btn amber">=</button>
+            <button onClick={this.clickEvent} name="AC" type="button" className="press-btn">AC</button>
+            <button onClick={this.clickEvent} name="" type="button" className="press-btn">+/-</button>
```
Comment (cynthiainga) [inline src/components/Calculator.js:37]: - [ ] Good job so far, but this button doesn’t work, you forgot to put the name. Please kindly fix that 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @tonnymuchui, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tonnymuchui** [comment]
Kindly check the branch you're in. because I can confirm from my pull
request is working properly
ᐧ

**Review by roshan-bajgain — APPROVED** [review]
Hi @tonnymuchui, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**